controllers/ticketingController.js

```diff
@@ -0,0 +1,13 @@
+const { catchAsync } = require('../utils/error');
+const ticketingService = require('../services/ticketingService');
+
+const getOptions = catchAsync(async (req, res) => {
+  try {
+    const options = await ticketingService.getOptions();
+    return res.status(200).json({ options });
+  } catch (err) {
+    throw err;
+  }
+});
```

Comment:
```suggestion
const getOptions = catchAsync(async (req, res) => {
    const options = await ticketingService.getOptions();
    
    return res.status(200).json({ options });
});
```
모듈화하여 만들어주신 catchAsync 함수를 사용하면 try catch 구문이 필요없습니다!